ansible/roles/ml-analytics-service/tasks/main.yml

```diff
@@ -21,9 +21,8 @@
       - openjdk-8-jdk
       - software-properties-common
       - python3-pip
-      - python3.6-dev
-      - python3.6-venv
-      - python-virtualenv
+      - python3.8-venv
```

Comment:
why are we specifying 3.8? won't 3-venv work?

Reply:
Changed @rjshrjndrn